### PR TITLE
Cortical Stacks

### PR DESCRIPTION
**Pull request by Andrew-Fall**
Stage one of #22 
Needs the system for player backups.

### CODE REVIEW

**Comment by biscuitWizard** [comment]
Don't forget to add your serialized variables to saved_vars.json at the root directory, especially the two in resleever.

File: code/game/machinery/resleever.dm [hunk 1]
```diff
@@ -0,0 +1,248 @@
...
+	var/obj/item/organ/internal/stack/lace = null
+
+	var/resleeving = 0
+	var/remaining = 0
+	var/timetosleeve = 120
+
+	var/occupant_name = null // Put in seperate var to prevent runtime.
+	var/lace_name = null
+
+/obj/machinery/resleever/New()
```
[the remark this answers is not in the document]
Comment (Andrew-Fall) [reply code/game/machinery/resleever.dm]: I think you clicked on a previous commit or something, that's old stuff I already removed.

File: code/game/objects/items/weapons/circuitboards/machinery/cloning.dm [hunk 2]
```diff
@@ -1,3 +1,19 @@
...
+	origin_tech = list(TECH_DATA = 3, TECH_BIO = 3)
+	req_components = list(
+							/obj/item/stack/cable_coil = 2,
+							/obj/item/weapon/stock_parts/scanning_module = 1,
+							/obj/item/weapon/stock_parts/manipulator = 3
+	)
+	additional_spawn_components = list(
+							/obj/item/weapon/stock_parts/console_screen = 1,
+							/obj/item/weapon/stock_parts/keyboard = 1,
+							/obj/item/weapon/stock_parts/power/apc/buildable = 1,
```
Comment (biscuitWizard) [inline code/game/objects/items/weapons/circuitboards/machinery/cloning.dm]: It takes an APC frame? That's super bizarre! but okay

Comment (Andrew-Fall) [reply code/game/objects/items/weapons/circuitboards/machinery/cloning.dm]: No, that is a power connector, most machines have it. 
image

**Pull request merged**